Doc/howto/enum.rst

```diff
@@ -1123,6 +1143,21 @@ The most interesting thing about enum members is that they are singletons.
 and then puts a custom :meth:`__new__` in place to ensure that no new ones are
 ever instantiated by returning only the existing member instances.
 
+Flag Members
+^^^^^^^^^^^^
+
+Flag members can be iterated over just like the Flag class, and only the
+canonical members will be returned.  For example,
+``list(Color) ==  [<Color.RED: 1>, <Color.GREEN: 2>, <Color.BLUE: 4>]``.
+(Note that ``BLACK``, ``PURPLE``, and ``WHITE`` do not show up.)
+
+Inverting a flag member does not return a negative value, but the
+corresponding positive value -- for example,
+``~Color.RED == <Color.GREEN|BLUE: 6>``.
```

Comment:
```suggestion
Inverting a flag member returns the corresponding positive value,
rather than a negative value --- for example::

    >>> ~Color.RED
    <Color.GREEN|BLUE: 6>
```

* Likewise, I set the example off in a code block to make it easier to read/parse and more explicit, and make it actually valid Python syntax
* The previously phrasing was pretty confusing/hard to read for me here (and could be particularly so for non-native speakers), so I reversed it.
* This should be an em dash (`—`/`---`), not an en dash (`–`/`--`).